test/directives/vector_directive1.f90

```diff
@@ -11,4 +11,5 @@ subroutine add(arr1,arr2,arr3,N)
     arr3(i) = arr1(i) - arr2(i)
   end do
 end subroutine
-! CHECK-NO-CLAUSE: F90-W-0602-No clause specified for the vector directive. Note: Only the always clause is supported.
+! CHECK-NO-CLAUSE-NOT: F90-S-0602
+! CHECK-NO-CLAUSE-NOT: F90-S-0603.
```

Comment:
Is the period at the end intentional?

Does this change mean that Flang now supports `!dir$ vector` (without the `always` clause)? If that's the case, you might also want to add `-emit-llvm` and check the IR to verify that vectorization happens.

Reply:
Yes `!dir$ vector` is now functional.
Added the llvm metadata + load/store test.
Fixed the period at the end of line 15.